src/index.html

```diff
@@ -22,6 +22,342 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo">
```

Comment:
The logo must have a link.
```suggestion
      <a 
        href="#"
        class="logo"
      >
```